src/views/Dashboard/Customer/EMR/UploadEMR.vue

```diff
@@ -326,14 +326,14 @@ export default {
     },
     titleRules() {
       return [
-        val => !!val || 'Title is Required',
-        val => (val && val.length <= 50) || 'Max 50 Character'
+        val => !!val || 'This field is required',
+        val => (val && val.length <= 50) || 'This field only allows 255 characters'
```

Comment:
The validation is <=50, but the text ist 255 characters. Please fix which one is true